packages/avatars/src/elements/Avatar.tsx

```diff
@@ -43,6 +47,24 @@ const AvatarComponent = forwardRef<HTMLElement, IAvatarProps>(
         {...other}
       >
         {Children.only(children)}
+        {(badge || status) && (
```

Comment:
This code is no longer able to show a badge of `'0'` as expected. If badge is defined, then it should be displayed.